### PR TITLE
Import wallet

### PR DESCRIPTION
**Pull request by duanyytop**
- Import wallet from seed
- Import wallet from keystore

### CODE REVIEW

File: packages/specs/api.d.ts [hunk 1]
```diff
@@ -210,6 +210,15 @@ export declare namespace Channel {
     type Response = SuccessResponse<boolean> | ErrorResponse
   }
 
+  namespace ImportKeystore {
+    interface Params {
+      name: string
+      keystorePath: string
+      password: string
+    }
+    type Response = SuccessResponse<boolean> | ErrorResponse
```
Comment (Keith-CY) [inline packages/specs/api.d.ts:219]: https://github.com/Keith-CY/keypering/pull/3/files#diff-f31bf3b2f4366eacb573d72bfa535201R119 indicates that a wallet profile is returned

File: packages/ui/src/components/KeystoreImporter/index.tsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+  const [submitting, setSubmitting] = useState(false)
+  const history = useHistory()
+
+  const handleInput = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const {
+      value,
+      files,
+      dataset: { fieldName },
+    } = e.target
+    dispatch({ fieldName: fieldName as keyof FormState, value: files ? files[0].path : value })
```
Comment (Keith-CY) [inline packages/ui/src/components/KeystoreImporter/index.tsx:40]: Is `files[0]` possible to be `undefined` here?

File: packages/ui/src/containers/Welcome/index.tsx [hunk 3]
```diff
@@ -2,19 +2,20 @@ import React from 'react'
 import { Link } from 'react-router-dom'
 import styles from './welcome.module.scss'
 
+const handleKeystore = () => {}
```
Comment (Keith-CY) [inline packages/ui/src/containers/Welcome/index.tsx]: It's not used anymore.

File: packages/ui/src/components/KeystoreImporter/index.tsx (hunk 2 above)
Comment (duanyytop) [reply packages/ui/src/components/KeystoreImporter/index.tsx]: Updated. https://github.com/Keith-CY/keypering/pull/3/commits/e30fbc85bf068b788695621553de783ed1e40586

File: packages/specs/api.d.ts (hunk 1 above)
Comment (duanyytop) [reply packages/specs/api.d.ts:219]: Updated. https://github.com/Keith-CY/keypering/pull/3/commits/e30fbc85bf068b788695621553de783ed1e40586

File: packages/ui/src/containers/Welcome/index.tsx (hunk 3 above)
Comment (duanyytop) [reply packages/ui/src/containers/Welcome/index.tsx]: Updated. https://github.com/Keith-CY/keypering/pull/3/commits/2c647ecdd50b480a5e5e26d4924f81b557ec1bc2

**Pull request merged**